setup.py

```diff
@@ -33,6 +33,7 @@
         "Source Code": "https://github.com/torchsynth/torchsynth/",
     },
     packages=find_packages(exclude=("tests", "examples")),
+    include_package_data=True,
```

Comment:
What is defined as package data? What gets rolled up into the pypi because of this?

Reply:
pretty much everything that is in the torchsynth package -- torchsynth/* 

I need to figure this one out though, I'd like to make it more specific. But it's also not working on travis.